Release 2.6.1

Techcable opened the pull request:
Bump version to 2.6.1

UTC by default (PR #30) is an important enough improvement/fix
that it warrants an immediate patch release.
    
This should not break any code because RFC 3339 requries
an explicit timezone for non-UTC times, and switching the timezone
should not break any conformant parser.

I also dropped the time/nested-values feature
because it is no longer needed.

Techcable committed:
Drop time/nested-values feature

It is not needed thanks to PR #30

Techcable committed:
Bump version to 2.6.1

UTC by default (#30) is an important enough improvement/fix
that it warrants an immediate patch release.

This should not break any code because RFC 3339 requries
an explicit timezone for non-UTC times, and switching the timezone
should not break any conformant parser.

jyn514 commented:
Hi @Techcable ! I appreciate the quick release, but it looks like this was only a github release and never made it to crates.io? I don't see it on https://crates.io/crates/slog-json.

Techcable commented:
@jyn514 Thanks for catching this (and contributing). This was a mistake on my part.

> only a github release

Actually it was missing from Github too. I just fixed that [here](https://github.com/slog-rs/json/releases/tag/v2.6.1)

>  never made it to crates.io? 

This is now fixed: https://crates.io/crates/slog-json/2.6.1

jyn514 commented:
Amazing, thanks again!
